doc/reference/autoinstall-reference.rst

```diff
@@ -215,9 +219,9 @@ network
 
 * **type:** Netplan-format mapping, see below
 * **default:** DHCP on interfaces named ``eth*`` or ``en*``
-* **can be interactive:** yes
+* **can be interactive:** true
 
-`Netplan-formatted <https://netplan.io/reference>`_ network configuration.
+`Netplan-formatted <https://netplan.readthedocs.io/en/stable/netplan-yaml/>`_ network configuration.
```

Comment:
I think it's sensible to leave this redirect in. Their RTD URL may change, but `netplan.io/reference` will likely be always point to the right thing.
```suggestion
`Netplan-formatted <https://netplan.io/reference>`_ network configuration.
```